search.html

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: Search JUGs
```

Comment:
```suggestion
title: Search
```
(I think it's clear were the user is, so is not need it to add the JUG word IMHO)

Reply:
Nice